dockerfile/rocm5.0-pytorch1.9.0.dockerfile

```diff
@@ -0,0 +1,134 @@
+FROM rocm/pytorch:rocm5.0_ubuntu18.04_py3.7_pytorch_1.9.0
+
+LABEL maintainer="SuperBench"
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
```

Comment:
In the old configurations, we have `wget -qO - http://repo.radeon.com/rocm/rocm.gpg.key | APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 apt-key add -`. Why do we need to remove it?